pkg/config/config.go

```diff
@@ -176,6 +176,7 @@ type Index struct {
 // Mattermost configuration to authentication and send notifications
 type Mattermost struct {
 	Enabled   bool
+	BotName   string
```

Comment:
@goir I believe, we would need to add yaml tag here `yaml: "botName"`. Could you please verify if the botName from config is getting mapped in the struct correctly?

Reply:
Hi @PrasadG193 you were right. had to add it, sorry i'm still learning go. But its working now. thank you